<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExposeWindow</class>
 <widget class="QDialog" name="ExposeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>561</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>421</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPlainTextEdit" name="information_plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Information on work</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="position_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Experiense</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="experiense_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salary</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="salary_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>560</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>86</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_vacancy_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View vacancy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="re_verification_button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Re sing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
